manual/vscode-extensions/vscode-ember-frost/vscode-ember-frost.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-frost</id>
+    <title>Ember Colorizer and Theme Extension</title>
+    <version>1.0.0</version>
+    <authors>Ciena Blue Planet</authors>
+    <owners>gep13 chocolatey</owners>
```

Comment:
Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)